lib/objectid.js

```diff
@@ -0,0 +1,5 @@
+var ObjectId = require('mongodb').ObjectId;
```

Comment:
My initial impl had our own ObjectId definition that delegated to mongo's object id, which we can move towards in the long term update. Will add it to the notes there

For the short term I will use the try/catch you suggested.